### PR TITLE
Add http notification

### LINKED ISSUE #121: Implement Alert send_http() notifications
It would be helpful if I could trigger an HTTP call on alert.

example: send_http("https://www.example.de/trigger?param1=...")

### PR DESCRIPTION
**Pull request by mohabusama**
- Add `notify_http`.
- Use POST only.
- Accept url, body, headers, params and including/excluding alert data.
- OAUTH2 supported.
- White list URLs or Allow All.

Resolve #121

### CODE REVIEW

File: zmon_worker_monitor/zmon_worker/notifications/http.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+               repeat=0):
+        urls = cls._config.get('notifications.http.whitelist.urls', [])
+        allow_any = cls._config.get('notifications.http.allow.all', False)
+        default_url = cls._config.get('notifications.http.default.url', None)
+
+        if isinstance(urls, basestring):
+            urls = urls.replace(' ', '').split(',')
+
+        if not url and not default_url:
+            raise NotificationError('url is required!')
```
Comment (hjacobs) [inline zmon_worker_monitor/zmon_worker/notifications/http.py]: "url" -> "URL" (as in the next exception)

File: zmon_worker_monitor/zmon_worker/notifications/http.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+        if not url:
+            url = default_url
+        elif not allow_any and url not in urls:
+            raise NotificationError('URL "{}" is not allowed. Please check worker white list URLs.'.format(url))
+
+        if not is_absolute_http_url(url):
+            raise NotificationError('Absolute url is required!')
```
Comment (hjacobs) [inline zmon_worker_monitor/zmon_worker/notifications/http.py]: "url" -> "URL" (as in the above exception)

File: zmon_worker_monitor/zmon_worker/notifications/http.py (hunk 1 above)
Comment (mohabusama) [inline zmon_worker_monitor/zmon_worker/notifications/http.py]: Done.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**